cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -30,6 +30,20 @@
 
 extern "C" void handle_vhpi_callback(const vhpiCbDataT *cb_data);
 
+VhpiArrayObjHdl::~VhpiArrayObjHdl()
+{
+    LOG_DEBUG("Releasing VhpiArrayObjHdl handle at %p\n", (void *)get_handle<vhpiHandleT>());
```

Comment:
nit: I think `vhpiHandleT` might already be `void*`. If it's not, then `static_cast<void*>(...)` would be the C++ spelling.

Fine as is though - this is only debug code anyway, and you fixed the bug already.

Reply:
From `vhpi_user.h`:

`typedef uint32_t *vhpiHandleT;`